libraries/render/src/render/CullTask.cpp

```diff
@@ -127,7 +127,10 @@ void FetchSpatialTree::run(const RenderContextPointer& renderContext, const Inpu
         // Octree selection!
         float threshold = 0.0f;
         if (queryFrustum.isPerspective()) {
-            threshold = args->_lodAngleHalfTan;
+            //TODO: It was:
+            // threshold = args->_lodFarAngleHalfTan;
```

Comment:
```suggestion
            // threshold = args->_lodAngleHalfTan;
```

Though I am not sure what this TODO is for. It seems as if this is done already and nothing needs to be done?

Reply:
This TODO means that before we had a fixed value for all the distances and right now LOD angle value changes with the distance so it assuming _lodFarAngleHalfTan seem to be obviously wrong but in tests it works. I'm marking this for the future if some unexpected problems pop up.

Reply:
Did you think to address the problem that we have with the current one about Material Entities that fail to be applied when very far cause material is always too small.

Example: have a big building very far but still visible (cause large), and have a material entity parented/applied on this building. when you are very far, (and never have loaded the material entity before), this one is not applied cause it is not loaded because it is too small.

Reply:
@AleziaKurdis Can you open an issue for this with an example on which I can test? It's a very important issue and I had this problem too before (it's not caused by this PR). We should discuss it in separate issue.

Reply:
It was already open but on under Vircadia...
So I moved the original bug in our repo:
https://github.com/overte-org/overte/issues/668

Reply:
I still don't see a difference between
"it was":
```cpp
threshold = args->_lodFarAngleHalfTan;
```
and "now should be dependent on distance":
```cpp
threshold = args->_lodFarAngleHalfTan;
```

If what we are doing here "seems obviously wrong", maybe we should just say that instead?